frontend/src/pages/audits/edit/findings/add/add.js

```diff
@@ -17,6 +17,9 @@ export default {
     },
```

Comment:
Please check quasar reference for tables as it already handles multiple selections: https://quasar.dev/vue-components/table#example--multiple-selection